mavlink/qgroundcontrol
# Add support for GNSS receiver resilience information

Add support for GNSS receiver resilience information

Description
-----------
Add GNSS signal authentication and interference indicators to the user interface and get the required data from MAVLink. The MAVLink code updates facts about the resilience information which are used in the QML files. The submodule still needs to be updated and the icons might need to be changed as well.

Context:
- https://github.com/mavlink/mavlink/pull/2110
- https://github.com/PX4/PX4-Autopilot/pull/23096

Test Steps
-----------
Install a version of PX4 autopilot or ArduPilot that supports resilience and use it with this PR.

Checklist:
----------


Related Issue
-----------
\/


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
